src/localization/en.js

```diff
@@ -264,6 +264,19 @@ export default {
       chartsHeader_v1:            `With the new pay, how could your benefits change?`,
       warningMessage_v1:          `This tool is in testing and these numbers might not be right. If they're not, we'd appreciate your feedback.`,
       submitFeedback_v1:          `Submit Feedback`,
+
+      benefitsTableTitle_v1:      `Changes`,
+      columnBenefit_v1:           `Benefit`,
+      columnCurrentBenefits_v1:   `Current Benefits`,
+      columnNewEstimate_v1:       `New Estimate`,
+      columnDifference_v1:        `Difference`,
+      rowSNAP_v1:                 `SNAP`,
+      rowSection8_v1:             `Section 8 Housing`,
+      rowTotalBenefits_v1:        `Total Benefits`,
+      rowIncome_v1:               `Income`,
+      rowNetTotal_v1:             `Net Total`,
+      perMonth_v1:                ` / month`,
+      dollarSign_v1:              `$`,
```

Comment:
I don't think the dollar sign needs to be a snippet - would it ever be different in another language? Same with the slash before "month".

I don't really have a problem with it, just found it slightly confusing.

Reply:
I think it's generally a good idea to have any string constant in translation files. That way the developer doesn't have to know if there's any language where it would need to be translated.

Now that you mention it though, I think the order can be different in different languages. I'm used to doing something like `translateString("$%1 / month").format(dollarAmount)`.

It's not immediately obvious to me how to do this with our setup (or even in javascript in general without a 3rd party library), and maybe that's better left to a separate PR?

Reply:
If we wanted to make it more flexible, we could treat each string like `$20 / month` as an interpolated snippet, so in English it would be like ``[ `$`, { name: `__monthlyAmount__` }, ` / month`]`` - if we had a way to delay interpolation from happening until the actual calculation of the amount occurred. (In this example, the interpolated component would just be `{ __monthlyAmount__: <span>{ amount }</span> }` and then you would call `interpolateText` on the array and that object within `BenefitsTable`.) 

Maybe this would work if we returned the item unchanged in `interpolateText` if the name wasn't found, rather than returning `null`.

Reply:
If we're going to get this complicated about that, maybe we should really consider adding a lib, like @pporfilio said. I think there was one out there that pluralized stuff, but I'm not sure if it was able to change the order of the text. There must be one that does let you do that, though. Then again, I'm not sure if it would play nice with the interpolation stuff. It does feel a bit like we're putting together Frankenstein right now.

If we were to do that kind of interpolation ourselves, wouldn't we need a way to interpolate in-place? @ethanbb : maybe your proposal would do that, but I wasn't sure how... I wonder if one of the other options I suggested for interpolation in #740 might have a good way to handle this. I've put it in the list for possible discussion/breakout.

Reply:
Hmm,. maybe `</T>` could have a `component` prop (which would be the snippet) and also children (which would be the changeable text), though I'm not quite sure how that would in a situation, for example, where there were multiple dynamic things going on...

Reply:
Yeah there are defnitely downsides and simplifications in what I suggested @knod, and I also feel like we should look for a more elegant way to do all this.

Not sure I understood your second comment. What's `<T>` in your example?

Another way to do this would be to have the snippet be a function of the dollar amount. We don't currently support snippets that are functions (and it would probably mess up the language property thing), but it would be cleaner than what I suggested.

Reply:
`<T>` is mentioned in #740 and might be discussed during the meeting. Functions for snippets was one of the ideas I was going to propose for interpolated snippets, but I'm not yet sure how we'd use them in what we have currently to be used for dynamic input. I have a thought about something like `var interp = function ({ snippets (with .langCode?), children, data/whatever? }) {...}` that's floating around at the moment...

Reply:
BTW the reason I had made two snippets for "$" and " / month" is because some languages do have a different order.  (We're not doing Thai, but Thai is one of those languages.)